llvm/lib/Transforms/Utils/Local.cpp

```diff
@@ -4220,8 +4220,11 @@ bool llvm::canReplaceOperandWithVariable(const Instruction *I, unsigned OpIdx) {
     return false;
 
   // Early exit.
-  if (!isa<Constant>(I->getOperand(OpIdx)))
+  if (!isa<Constant>(I->getOperand(OpIdx))) {
```

Comment:
```suggestion
  if (!isa<Constant, InlineAsm>(I->getOperand(OpIdx))) {
```
Could we do something like this instead?

Reply:
That's a nice way to do that, thanks. Seems to be working locally.